dependencies/clean.sh

```diff
@@ -0,0 +1,6 @@
+sudo -u postgres psql -c "drop database django_acmgeneral"
```

Comment:
This file should not have been committed. I think it is my fault, but can you remove it.